README.md

```diff
@@ -74,12 +74,52 @@ highlights the enclosed text in yellow
     The VAT rate is *20%*
     {:/highlight-answer}
 
-creates a large pink highlight box with optional preamble text and giant text denoted with `**`
+creates a large highlight box with optional preamble text and giant text denoted with `**`
 
     <div class="highlight-answer">
     <p>The VAT rate is <em>20%</em></p>
     </div>
 
+### Statistic headline
+
+Used in HTML publications.
+
+Statistic headlines highlight important numbers in content. Displays a statistic as a large number with a description. The statistic and description must make sense when read aloud. The giant number is denoted with `**`.
```

Comment:
> The giant number is denoted with `**`.

I can see that is consistent with the `Answer` section, but why is it not something like "wrapped with *'s"?